tests/layer_tests/pytorch_tests/test_leaky_relu.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestLeakyRelu(PytorchLayerTest):
+    def _prepare_input(self):
+        import numpy as np
+        return (np.random.randn(1, 3, 224, 224).astype(np.float32),)
+
+    def create_model(self, alpha, inplace):
+        import torch
+        import torch.nn.functional as F
+
+        class aten_leaky_relu(torch.nn.Module):
+            def __init__(self, alpha, inplace):
+                super(aten_leaky_relu, self).__init__()
+                self.alpha = alpha
+                self.inplace = inplace
+
+            def forward(self, x):
+                return F.leaky_relu(x, self.alpha, inplace=self.inplace)
+            
+        ref_net = None
+
+        return aten_leaky_relu(alpha, inplace), ref_net, "aten::leaky_relu" if not inplace else "aten::leaky_relu_"
+
+
+    @pytest.mark.parametrize("alpha,inplace", [(0.01, True), (0.01, False), (-0.01, True), (-0.01, False)])
```

Comment:
`0.01` is default value for `leaky_relu`, could you add non default positive case?

Reply:
added